doc/src/modules/solvers/solveset.rst

```diff
@@ -398,7 +398,7 @@ How does ``solveset`` ensure that it is not returning any wrong solution?
     >>> from sympy import symbols, S, pprint, solveset
     >>> x, n = symbols('x, n')
     >>> pprint(solveset(abs(x) - n, x, domain=S.Reals), use_unicode=True)
-    [0, ∞) ∩ {n}
+    {x | x ∊ {-n, n} ∧ ({{n} ∈ [0, ∞)})}
```

Comment:
- Yes.  The are some redundant parantheses here.
I wrapped the `Contains` object with a `FiniteSet` so as to facilitate Union operation on more `Contains` objects if there be such a situation. This contributes to an overhead curly brace of the output. We can mitigate this by adding more than one condition to the `ConditionSet` object.
Would passing an `And` object here be a good solution to this issue ?
-  I used a FiniteSet object to represent the conditioned symbols so as to give a compact output.
I was having an issue with using the args of the `FiniteSet`
For example:
When using the `FiniteSet.args`
Output : ` (n,) ∈ [0, ∞)`
Instead of `(n) ∈ [0, ∞) `  
Also, I want to know which of the following results should we prefer ?
`((b, a - b, a + b) ∈ [0, ∞))` or `(b, a - b, a + b ∈ [0, ∞)) or ` (b ∈ [0, ∞)) ∧ (a - b ∈ [0, ∞)) ∧ (a + b ∈ [0, ∞))`
- The `∈` (epsilon) in the output is due to the printer of `Contains` object.
I think this might be a bug.  

@asmeurer Thanks for this.
I Hope, I have